architecture-docs/firebase-architecture.md

```diff
@@ -0,0 +1,12 @@
+The Ground platform uses Firebase for many backend and utility operations, including authentication and storage. 
+
+Currently, Ground uses the following Firebase features:
+
+- [**Cloud Firestore**](https://firebase.google.com/docs/firestore) is used for storing Ground observation data. Data in Firestore is organized into collections and documents. For more information on how Ground data is structured, see [Cloud Firestore Representation](https://github.com/google/ground-platform/wiki/Cloud-Firestore-Representation).
```

Comment:
-> how Ground data is represented in Cloud Firestore

Reply:
Done.